README.md

```
@@ -230,36 +254,59 @@ Features: 6 filter tabs, 4 sort modes, grouped/flat view, lazy-loaded previews,
 
 ## Project Structure
 
+### Source repo (career-ops)
+
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)